src/pages/resultsView/mutation/Mutations.tsx

```diff
@@ -92,7 +94,8 @@ export default class Mutations extends React.Component<IMutationsPageProps, {}>
                             mutationCountCache={this.props.store.mutationCountCache}
                             pdbHeaderCache={this.props.store.pdbHeaderCache}
                             myCancerGenomeData={this.props.store.myCancerGenomeData}
-                            config={AppConfig}
+                            config={AppConfig.serverConfig}
+                            userEmailAddress={this.props.appStore.userName!}
```

Comment:
@onursumer why is userEmailAddress necessary here?   is it ok that this is userName?  i'm not sure it will always be email.     
Also, FYI, a user can now log in without reload of the page.